ui/pages/swaps/transaction-settings/transaction-settings.js

```diff
@@ -30,12 +30,12 @@ import {
   BannerAlert,
   Modal,
   ModalOverlay,
-  ModalContent,
-  ModalHeader,
   ButtonPrimary,
   ButtonLink,
   ButtonLinkSize,
 } from '../../../components/component-library';
+import { ModalContent } from '../../../components/component-library/modal-content/deprecated';
+import { ModalHeader } from '../../../components/component-library/modal-header/deprecated';
 import { setSwapsErrorKey } from '../../../store/actions';
 import { getSwapsErrorKey } from '../../../ducks/swaps/swaps';
 
```

Comment:
Screenshots showing no visual changes to UI

### Before
<img width="1512" alt="Screenshot 2023-12-10 at 5 11 30 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/4b709e95-8373-44b7-bfd5-f5979231305c">

### After
<img width="1512" alt="Screenshot 2023-12-10 at 5 11 42 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/a6cae6a8-ad8c-4bdc-a388-36ef92b83c9b">